aws_lambda_powertools/utilities/parser/envelopes/vpc_lattice.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from typing import Any, Dict, Optional, Type, Union
+
+from ..models import VpcLatticeModel
+from ..types import Model
+from .base import BaseEnvelope
+
+logger = logging.getLogger(__name__)
+
+
+class VpcLatticeEnvelope(BaseEnvelope):
+    """Amazon VPC Lattice envelope to extract data within body key"""
+
+    def parse(self, data: Optional[Union[Dict[str, Any], Any]], model: Type[Model]) -> Optional[Model]:
+        """Parses data found with model provided
+
+        Parameters
+        ----------
+        data : Dict
+            Lambda event to be parsed
+        model : Type[Model]
+            Data model provided to parse after extracting data using envelope
+
+        Returns
+        -------
+        Any
+            Parsed detail payload with model provided
+        """
+        logger.debug(f"Parsing incoming data with VPC Lattice model {VpcLatticeModel}")
+        parsed_envelope: VpcLatticeModel = VpcLatticeModel.parse_obj(data)
+        print(parsed_envelope.body)
```

Comment:
```suggestion
```

Should we remove this?

Reply:
Sorry, Ruben! I was debugging and forgot this!